doc/index.rst

```diff
@@ -43,99 +87,81 @@ automatic differentiation, and optimization of hybrid quantum-classical computat
 
 
 - *Follow the gradient*.
-  Built-in **automatic differentiation** of quantum circuits
+  Built-in **automatic differentiation** of quantum circuits.
 
 ..
 
 - *Best of both worlds*.
-  Support for **hybrid quantum and classical** models
+  Support for **hybrid quantum and classical** models.
 
 ..
 
 - *Batteries included*.
-  Provides **optimization and machine learning** tools
+  Provides **optimization and machine learning** tools.
 
 ..
 
 - *Device independent*.
-  The same quantum circuit model can be **run on different backends**
+  The same quantum circuit model can be **run on different backends**. Install
+  `plugins <https://pennylane.ai/plugins.html>`_ to run your computational circuits
+  on even more devices, including **Strawberry Fields**, **Rigetti Forest**, **ProjectQ**,
+  **Microsoft QDK**, and **IBM Q**.
 
 ..
 
 - *Compatible with existing machine learning libraries*.
   Quantum circuits can be set up to interface with either **NumPy**, **PyTorch**, or **TensorFlow**,
   allowing hybrid CPU-GPU-QPU computations.
 
-..
 
-- *Large plugin ecosystem*.
-  Install plugins to run your computational circuits on more devices, including **Strawberry Fields**, **Rigetti Forest**, **ProjectQ**, **Microsoft QDK**, and **IBM Q**
 
-:html:`<h4>Available plugins</h4>`
+Getting started
+---------------
 
-* `PennyLane-SF <https://github.com/XanaduAI/pennylane-sf>`__: Supports integration with `Strawberry Fields <https://github.com/XanaduAI/strawberryfields>`__, a full-stack Python library for simulating continuous variable (CV) quantum optical circuits.
+For getting started with PennyLane, check out some of the
+`key concepts <https://pennylane.ai/qml/concepts.html>`_ behind quantum machine
+learning, before moving on to some `introductory tutorials <https://pennylane.ai/qml/beginner.html>`_.
 
-..
+Then, take a deeper dive into quantum machine learning by
+exploring cutting-edge algorithms using PennyLane and near-term quantum hardware,
+with our collection of
+`QML tutorials <https://pennylane.ai/qml/implementations.html>`_.
 
-* `PennyLane-Forest <https://github.com/rigetti/pennylane-forest>`_: Supports integration with `PyQuil <https://github.com/rigetti/pyquil>`__, the `Rigetti Forest SDK <https://www.rigetti.com/forest>`__, and the `Rigetti QCS <https://www.rigetti.com/qcs>`__, an open-source quantum computation framework by Rigetti. Provides device support for the Quantum Virtual Machine (QVM) and Quantum Processing Units (QPUs) hardware devices.
+You can also check out our `documentation <https://pennylane.readthedocs.io>`_ for
```

Comment:
This is a self-hyperlink :laughing:

Reply:
Replaced this with a new paragraph, fixing the 'self' links:

```rest
You can also check out the :doc:`Using PennyLane <introduction/pennylane>` section for
more details on the :doc:`quantum operations <introduction/operations>`, and to explore
the available :doc:`optimization tools <introduction/optimizers>` provided by PennyLane,
and detailed guides on :doc:`how to write your own <introduction/plugins>`
PennyLane-compatible quantum device.
```

Reply:
Note: Even though `:doc:` links depend on the relative location/filename, I prefer `:doc:` links over `:ref:` links for page links (not section links) solely because `:ref:` causes the page to scroll slightly,